src/main/java/uk/gov/dhsc/htbhf/smartstub/controller/CardServicesController.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dhsc.htbhf.smartstub.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.dhsc.htbhf.smartstub.model.*;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/v1/cards")
+@Slf4j
+public class CardServicesController {
+
+    @PostMapping
+    public CreateCardResponse createCard(@RequestBody CardRequestDTO cardRequestDTO) {
+        CreateCardResponse createCardResponse = CreateCardResponse.builder()
+                .cardAccountId(UUID.randomUUID().toString())
```

Comment:
Just had a thought that will make private beta easier. Can we add a prefix here please: `"stub-" + UUID.randomUUID()`. This will make it possible to identify card ids from the stub. We'll need to do the same thing with payments.

Reply:
Done.